Frameworks/Starboard/EbrFile.cpp

```diff
@@ -163,14 +172,50 @@ bool EbrUnlink(const char* path) {
 
 #define FSROOT "."
 #define mkdir _mkdir
-char g_WritableFolder[2048] = ".";
 
-void EbrSetWritableFolder(const char* folder) {
-    strcpy_s(g_WritableFolder, folder);
+Wrappers::HString _getAppDataPath() {
+    Wrappers::HString toReturn;
+    ComPtr<IStorageFolder> folder;
+    ComPtr<IApplicationDataStatics> applicationDataStatics;
+    ComPtr<IApplicationData> applicationData;
+    ComPtr<IStorageItem> storageItem;
+
+    if (FAILED(GetActivationFactory(Wrappers::HStringReference(RuntimeClass_Windows_Storage_ApplicationData).Get(),
+                                    &applicationDataStatics))) {
+        return toReturn;
+    }
+
+    if (FAILED(applicationDataStatics->get_Current(&applicationData))) {
+        return toReturn;
+    }
```

Comment:
I found it and yes we can use it but we don't have a macro for that. I don't really want to make a new one. If you are super adamant, I can look at doing that.

---
In reply to: [98271548](https://github.com/Microsoft/WinObjC/pull/1849#discussion_r98271548) [](ancestors = 98271548)